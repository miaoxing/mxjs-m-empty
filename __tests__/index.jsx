import Empty from '..';
import {render} from '@testing-library/react';

describe('Empty', () => {
  test('basic', () => {
    const {container} = render(<Empty/>);
    expect(container).toMatchSnapshot();
  });
});
