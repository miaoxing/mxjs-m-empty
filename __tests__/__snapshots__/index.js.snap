// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty basic 1`] = `
<div>
  <taro-view-core
    style="flex-direction: column; color: rgb(113, 113, 122); justify-content: center; align-items: center; display: flex;"
  >
    <taro-view-core
      class="mx-empty-image"
      style="width: 128px; height: 82px;"
    />
    <taro-view-core>
      暂无数据
    </taro-view-core>
    <taro-view-core />
  </taro-view-core>
</div>
`;
